src/sentry/static/sentry/app/views/projectAlertSettings.jsx

```diff
@@ -180,13 +182,17 @@ export default class ProjectAlertSettings extends AsyncView {
     let {organization} = this.props;
     return (
       <div>
-        <a
-          href={`/${orgId}/${projectId}/settings/alerts/rules/new/`}
-          className="btn pull-right btn-primary btn-sm">
-          <span className="icon-plus" />
-          {t('New Alert Rule')}
-        </a>
-        <h2>{t('Alerts')}</h2>
+        <SpreadLayout style={{marginBottom: 20}}>
+          <h2 style={{margin: 0}}>{t('Alerts')}</h2>
+          <Button
```

Comment:
semantically, this seems kind of like a Link component (but it doesn't do the styles and stuff you want for this). not sure what i'm getting at but it might make sense to codify buttons that have and onclick vs buttons with an href sometime? because, for instance, these won't try to use react router and I could imagine a case where someone accidentally causes a real page load to a react route

Reply:
I get what you're trying to say - I think this gets solved when we port the rest of our settings from django so we will always be using react-router routes. Currently we still have the problem of having to figure out whether something is in django or react anyway. I'm thinking we just leave button alone for now, but I did submit #6200 to use for external links.

Reply:
agreed